pkg/apis/core/type_landscaperdeployment.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2021 "SAP SE or an SAP affiliate company and Gardener contributors"
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package core
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// The LandscaperDeployment is created to define a deployment of the Landscaper Service.
+type LandscaperDeployment struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec contains the specification for the LandscaperDeployment
+	Spec LandscaperDeploymentSpec `json:"spec"`
+
+	// Status contains the status of the LandscaperDeployment.
+	// +optional
+	Status LandscaperDeploymentStatus `json:"status"`
+}
+
+// LandscaperDeploymentSpec contains the specification for a LandscaperDeployment.
+type LandscaperDeploymentSpec struct {
```

Comment:
maybe one configuration option would be the default deployers that are installed?

Reply:
👍